rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2379,6 +2355,17 @@ public Coin getLockingCap() {
         return this.provider.getLockingCap();
     }
 
+    /**
+     * Returns the redeemScript of the current active federation
+     * @return Returns the redeemScript of the current active federation
+     */
+    public Script getActivePowpegRedeemScript() {
+        if (activations.isActive(ConsensusRule.RSKIP293)) {
+            return getActiveFederation().getRedeemScript();
+        }
+        return new Script(new byte[]{});
```

Comment:
```suggestion
        return activations.isActive(ConsensusRule.RSKIP293) ? 
                   getActiveFederation().getRedeemScript() :
                   new Script(new byte[]{});
```
A shorter version if you prefer